src/test/java/uk/gov/hmcts/reform/managecase/service/noc/PrepareNoCServiceTest.java

```diff
@@ -0,0 +1,366 @@
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", CASE_TYPE, caseData,
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("The Organisation of the solicitor is not recorded in an Org policy on the case.")
+            );
+        }
+
+        @Test
+        @DisplayName("should error when there is ongoing NoCRequest")
+        void shouldErrorWhenThereIsOngoingNoCRequest() {
+            mockUserCaseRolesForSolicitor();
+
+            Map<String, JsonNode> caseData = new HashMap<>();
+            orgPolicies.forEach((key, value) -> caseData.put(key, organisationPolicyToJsonNode(value)));
+            caseData.put(COR_FIELD_NAME, changeOrganisationRequestToJsonNode(ongoingChangeOrganisationRequest()));
+
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", CASE_TYPE, caseData,
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("There is an ongoing NoCRequest.")
+            );
+        }
+
+        @Test
+        @DisplayName("should error when Jurisdiction is missing")
+        void shouldErrorWhenMissingJurisdiction() {
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, null, "", CASE_TYPE, new HashMap<>(),
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("Jurisdiction cannot be blank.")
+            );
+        }
+
+        @Test
+        @DisplayName("should error when CaseType is missing")
+        void shouldErrorWhenMissingCaseType() {
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", null, new HashMap<>(),
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("CaseType cannot be blank.")
```

Comment:
If string should match validation error then import and match on constant defined in *src/main/java/uk/gov/hmcts/reform/managecase/api/errorhandling/ValidationError.java*.